tests/foreman/ui/test_discoveryrule.py

```diff
@@ -66,11 +66,13 @@ def manager_user(manager_loc, module_location, module_org):
     return manager_user
 
 
-@pytest.fixture
-def reader_user(module_location, module_org):
+@pytest.fixture(scope='module')
+def reader_user(module_location, module_org, module_target_sat):
     password = gen_string('alphanumeric')
-    reader_role = entities.Role().search(query={'search': 'name="Discovery Reader"'})[0]
-    reader_user = entities.User(
+    reader_role = module_target_sat.api.Role().search(query={'search': 'name="Discovery Reader"'})[
+        0
+    ]
```

Comment:
This looks odd to me. Did the linter do this?

Reply:
I will check it.

Reply:
yes I believe it's because of the  checks

Reply:
Non-blocking. If you want to make it look better, you could try splitting lines after the `search` starts.

Reply:
Can we have some short variable name here so this can be adjusted in single line?